pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -338,4 +338,20 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.TimeWindow.End).To(HaveSuffix("0000+0000"))
 		})
 	})
+
+	Describe("#SetDefaults_Worker", func() {
+		var obj *Worker
+
+		BeforeEach(func() {
+			obj = &Worker{}
+		})
+
+		It("should set the allowSystemComponents field", func() {
+			obj.AllowSystemComponents = nil
+
+			SetDefaults_Worker(obj)
+
+			Expect(obj.AllowSystemComponents).To(Equal(&DefaultWorkerAllowSystemComponents))
```

Comment:
```suggestion
			Expect(obj.AllowSystemComponents).To(PointTo(BeTrue()))
```
Same for the other API version.

Reply:
Changed